### PR TITLE
Coaching Dashboard Starting Point

### PR DESCRIPTION
**Pull request by LuckyHariharan**
In this PR:

- Created ProfileDetailCard type and component
- Used dummy data to display enrolled users
- Established a starting point to build forward 

https://www.loom.com/share/ba6d3f3b7ac849d8bf8f80e21f73c140

### CODE REVIEW

**Review by vithushan19 — COMMENTED** [review]
the cards look great! I would create a new page called `coaches/dashboard` and move the cards there. 

The page doesn't have to be accessible through the sidebar for now, but we would know internally that we can access the page through the URL directly.

File: components/coding/studentPortal/ProfileDetailCard.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <img className="rounded-full border-2 w-20 " src={avatar} />
+        <p className=" text-black"> {name}</p>
+        <p className="">join date: {joinDate}</p>
+        <p className="">badges {badges}</p>
+      </div>
+      <div className="flex flex-col gap-4">
+        <p className="underline-offset-8">current focus: {currentFocus}</p>
+        <p>next goal: {nextGoal}</p>
+      </div>
+      {/* <a href={starterFile}>
```
Comment (iamnotreddy) [inline components/coding/studentPortal/ProfileDetailCard.tsx:33]: try not to commit commented code

**Review by iamnotreddy — APPROVED** [review]

File: pages/coaching/coachingdashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      nextGoal: "learning javascript syntax",
+    },
+  ];
+  return (
+    <div className="flex flex-col p-4 m-4 overflow-auto bg-scroll">
+      <div>
+        <div className="mb-8 text-3xl">Coaching Dashboard</div>
+        <h2 className="mb-4">Enrolled Students</h2>
+        <div className="grid grid-cols-1 gap-4 sm:grid-cols-2">
+          {profileData.map((it) => (
```
Comment (kwiedenhaupt) [inline pages/coaching/coachingdashboard.tsx:58]: great way to use the map function

File: components/coding/studentPortal/ProfileDetailCard.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  return (
+    <div className="bg-gray-900 text-sm w-60 p-6 space-y-8 rounded-md ">
+      <div className="flex flex-col grow items-center text-center gap-4 font-bold  ">
+        <img className="rounded-full border-2 w-20 " src={avatar} />
+        <p className=" text-black"> {name}</p>
+        <p className="">join date: {joinDate}</p>
+        <p className="">badges {badges}</p>
+      </div>
+      <div className="flex flex-col gap-4">
+        <p className="underline-offset-8">current focus: {currentFocus}</p>
```
Comment (kwiedenhaupt) [inline components/coding/studentPortal/ProfileDetailCard.tsx:30]: underline-offset-8 makes it look a lot cleaner. cool tailwind find.

**Review by kwiedenhaupt — COMMENTED** [review]
great job setting up the profile detail card and rendering it on the page. great launch!

File: pages/coaching/coachingdashboard.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+      name: "Jessica Yale",
+      joinDate: "2021-11-7",
+      badges: "2/35",
+      currentFocus: "using React to build web applications",
+      nextGoal: "learning Redux",
+    },
+    {
+      avatar:
+        "https://images.unsplash.com/photo-1584999734482-0361aecad844?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=580&q=80",
+      name: "Mark Duckerberg",
```
Comment (iamnotreddy) [inline pages/coaching/coachingdashboard.tsx:51]: 🐤

**Review by iamnotreddy — APPROVED** [review]
nice work on this - looks ready to merge

**Pull request merged**